nuclear_simulator/data_gen/runners/maintenance_scenario_runner.py

```
@@ -0,0 +1,1498 @@
+                print(f"   ⏱️ Time step: {time_step_minutes} minutes")
+                print(f"   🔧 Maintenance system: {'Enabled' if self.maintenance_system else 'Disabled'}")
+        
+        except Exception as e:
+            print(f"   ❌ Error initializing simulator: {e}")
+            raise
+    
+    def _initialize_secondary_systems(self):
+        """Initialize secondary systems from configuration"""
+        try:
+            # Get secondary system configuration
+            secondary_config = self.config.get('secondary_system', {})
```

Ruff (0.11.9)

Local variable `secondary_config` is assigned to but never used

Remove assignment to unused variable `secondary_config`

(F841)